src/Notes/WC_Admin_Notes_Choose_Niche.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * WooCommerce Admin: Choose a niche note.
+ *
+ * Adds a note to show the client how to choose a niche for their store.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Choose_Niche.
+ */
+class WC_Admin_Notes_Choose_Niche {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-choose-niche';
+
+	/**
+	 * Get the note.
+	 */
+	public static function get_note() {
+		$onboarding_profile = get_option( 'woocommerce_onboarding_profile', array() );
+		if (
+			! isset( $onboarding_profile['setup_client'] ) ||
+			! isset( $onboarding_profile['product_count'] ) ||
+			! isset( $onboarding_profile['revenue'] )
```

Comment:
This check prevents the "OR" logic that Pedro describes. We should be able to select "I don't have any products" AND "I'm not selling anywhere else" and see the notice.

Reply:
Nice catch! Thank you @jeffstieler.
This is solved in the commit 7056ecb